components/ui/icons/src/main/res/drawable/mozac_ic_close_20.xml

```diff
@@ -5,9 +5,9 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="20dp"
     android:height="20dp"
-    android:viewportWidth="20.0"
-    android:viewportHeight="20.0">
-    <path
-        android:fillColor="@color/mozac_ui_icons_fill"
-        android:pathData="M11.1081 9.77607L15.8171 5.06707C15.9264 4.94825 15.9855 4.79181 15.9822 4.63042C15.9789 4.46902 15.9133 4.31516 15.7992 4.20095C15.6852 4.08674 15.5314 4.02101 15.37 4.0175C15.2086 4.01398 15.0521 4.07294 14.9331 4.18207L10.2441 8.87107H9.75613L5.06713 4.18307C5.00902 4.12503 4.94005 4.079 4.86415 4.04761C4.78825 4.01622 4.70691 4.00009 4.62478 4.00014C4.54264 4.00018 4.46132 4.01641 4.38546 4.04788C4.3096 4.07936 4.24068 4.12546 4.18263 4.18357C4.12459 4.24168 4.07856 4.31066 4.04717 4.38656C4.01578 4.46246 3.99965 4.54379 3.99969 4.62593C3.99979 4.7918 4.06577 4.95085 4.18313 5.06807L8.87013 9.75407V10.2451L4.18313 14.9321C4.07387 15.0509 4.01473 15.2073 4.01806 15.3687C4.02139 15.5301 4.08695 15.684 4.20103 15.7982C4.31511 15.9124 4.4689 15.9781 4.63029 15.9817C4.79167 15.9852 4.94818 15.9262 5.06713 15.8171L9.75413 11.1301H10.2451L14.9321 15.8171C15.0496 15.9341 15.2088 15.9999 15.3746 15.9999C15.5405 15.9999 15.6996 15.9341 15.8171 15.8171C15.9344 15.6997 16.0003 15.5405 16.0003 15.3746C16.0003 15.2086 15.9344 15.0495 15.8171 14.9321L11.1081 10.2231V9.77607Z" />
+    android:viewportWidth="20"
+    android:viewportHeight="20">
+  <path
+      android:fillColor="@color/mozac_ui_icons_fill"
+      android:pathData="M11.108,9.776L15.817,5.067C15.926,4.948 15.986,4.792 15.982,4.63C15.979,4.469 15.913,4.315 15.799,4.201C15.685,4.087 15.531,4.021 15.37,4.017C15.209,4.014 15.052,4.073 14.933,4.182L10.244,8.871H9.756L5.067,4.183C5.009,4.125 4.94,4.079 4.864,4.048C4.788,4.016 4.707,4 4.625,4C4.543,4 4.461,4.016 4.385,4.048C4.31,4.079 4.241,4.125 4.183,4.184C4.125,4.242 4.079,4.311 4.047,4.387C4.016,4.462 4,4.544 4,4.626C4,4.792 4.066,4.951 4.183,5.068L8.87,9.754V10.245L4.183,14.932C4.074,15.051 4.015,15.207 4.018,15.369C4.021,15.53 4.087,15.684 4.201,15.798C4.315,15.912 4.469,15.978 4.63,15.982C4.792,15.985 4.948,15.926 5.067,15.817L9.754,11.13H10.245L14.932,15.817C15.05,15.934 15.209,16 15.375,16C15.54,16 15.7,15.934 15.817,15.817C15.934,15.7 16,15.54 16,15.375C16,15.209 15.934,15.05 15.817,14.932L11.108,10.223V9.776Z" />
```

Comment:
Can you also this file change? I still see the AS performance warning for very long vector path with this change. So, I think we should just accept it as is for the time being while seeing if UX can improve on their side.

Reply:
It's the path of a [file](https://www.figma.com/file/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?node-id=844%3A1210) we get from figma, not optimized.